linters/markdown-link-check/parse.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+"""
+Parser for markdown-link-check stdout.
+
+markdown-link-check output looks like this
+
+```
+[✖] https://docs.trunk.io/docs/compatibility → Status: 404
+[✖] /linters/stringslint/stringslint.test.ts → Status: 400
+[✖] ../linters/sqlfluff/test/sqlfluff_test.ts → Status: 400
+```
+"""
+
+import json
+import re
+import sys
+
+
+def to_result_sarif(
+    path: str, line_number: int, column_number: int, rule_id: str, message: str
+):
+    return {
+        "level": "error",
+        "locations": [
+            {
+                "physicalLocation": {
+                    "artifactLocation": {
+                        "uri": path,
+                    },
+                    "region": {
+                        "startColumn": column_number,
+                        "startLine": line_number,
+                    },
+                }
+            }
+        ],
+        "message": {
+            "text": message,
+        },
+        "ruleId": rule_id,
+    }
+
+
+def main(argv):
+    results = []
+
+    # output = sys.stdin
+
+    # issues = output.split()
```

Comment:
remove, and add comment saying that we can't use our built-in regex output type because it doesn't report line numbers

Reply:
done